.travis.yml

```diff
@@ -1,4 +1,3 @@
-sudo: false
```

Comment:
is this the default now - or are we running on the slower VMs now?
I think the main point was that sudo:false gives us a speed boost because TravisCI uses faster containers instead of a full VM.

Reply:
I think so, it has been deprecated (see https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration)

Reply:
oh interesting! 👍